schemas/iso19115-3.2018/src/main/plugin/iso19115-3.2018/formatter/dcat/dcat-core-access-and-use.xsl

```diff
@@ -0,0 +1,159 @@
+                select="false()"/>
+
+  <!--
+  RDF Property:	dcterms:accessRights
+  Definition:	Information about who can access the resource or an indication of its security status.
+  Range: dcterms:RightsStatement
+  = First constraints of mdb:identificationInfo/*/mri:resourceConstraints/*[mco:accessConstraints] (then dct:rights)
+
+  RDF Property:	dcterms:license
+  Definition:	A legal document under which the resource is made available.
+  Range:	dcterms:LicenseDocument
+  Usage note:	Information about licenses and rights MAY be provided for the Resource. See also guidance at 9. License and rights statements.
+  = First useLimitation or constraints of mdb:identificationInfo/*/mri:resourceConstraints/*[mco:useConstraints] (then dct:rights)
+
+  RDF Property:	dcterms:rights
+  Definition:	A statement that concerns all rights not addressed with dcterms:license or dcterms:accessRights, such as copyright statements.
+  Range:	dcterms:RightsStatement
+  Usage note:	Information about licenses and rights MAY be provided for the Resource. See also guidance at 9. License and rights statements.
+  -->
+  <xsl:template mode="iso19115-3-to-dcat"
+                match="mdb:identificationInfo/*/mri:resourceConstraints/*[mco:accessConstraints]">
+    <xsl:if test="count(../preceding-sibling::mri:resourceConstraints/*[mco:accessConstraints]) = 0">
+      <xsl:for-each select="../../mri:resourceConstraints/*[mco:accessConstraints]/mco:otherConstraints">
+        <xsl:if test="position() = 1 or ($isPreservingAllResourceConstraints and position() > 1)">
+          <xsl:element name="{if (position() = 1) then 'dct:accessRights' else 'dct:rights'}">
+            <dct:RightsStatement>
+              <xsl:choose>
+                <xsl:when test="gcx:Anchor/@xlink:href">
+                  <xsl:attribute name="rdf:about" select="gcx:Anchor/@xlink:href"/>
+                </xsl:when>
+                <xsl:otherwise>
+                  <xsl:call-template name="rdf-localised">
+                    <xsl:with-param name="nodeName" select="'dct:description'"/>
+                  </xsl:call-template>
+                </xsl:otherwise>
+              </xsl:choose>
+            </dct:RightsStatement>
+          </xsl:element>
+        </xsl:if>
+      </xsl:for-each>
+    </xsl:if>
+  </xsl:template>
+
+
+  <xsl:template mode="iso19115-3-to-dcat"
+                name="iso19115-3-to-dcat-license"
+                match="mdb:identificationInfo/*/mri:resourceConstraints/*[mco:useConstraints]">
+    <xsl:if test="count(../preceding-sibling::mri:resourceConstraints/*[mco:useConstraints]) = 0">
+      <xsl:variable name="useConstraints"
+                    as="node()*">
+        <xsl:copy-of select="ancestor::mri:resourceConstraints/*[mco:useConstraints]/mco:otherConstraints"/>
+        <xsl:copy-of select="ancestor::mri:resourceConstraints/*[mco:useConstraints]/mco:useLimitation"/>
+      </xsl:variable>
+      <xsl:for-each select="$useConstraints">
```

Comment:
Here we need to adapt following GeoDCAT-AP 3 revision

* `dct:license` and `dct:accessRights` will be following HVD - only machine readable so only IRI. If the license is not found in the vocabulary then a `dct:rights` is created.
* Cardinality of `dct:rights` is relaxed from `0..1` to `0..n`